spyder/plugins/projects/plugin.py

```diff
@@ -203,6 +206,16 @@ def on_main_menu_available(self):
             menu_id=ApplicationMenus.Projects,
             section=ProjectsMenuSections.Extras)
 
+    @on_plugin_available(plugin=Plugins.Switcher)
+    def on_switcher_available(self):
+        # Connect to switcher
+        self._switcher = self.main.switcher
+        self._switcher.sig_mode_selected.connect(self.handle_switcher_modes)
+        self._switcher.sig_item_selected.connect(
+            self.handle_switcher_selection)
+        self._switcher.sig_search_text_available.connect(
+            self.handle_switcher_results)
```

Comment:
```suggestion
        self._switcher.sig_mode_selected.connect(self._handle_switcher_modes)
        self._switcher.sig_item_selected.connect(
            self._handle_switcher_selection)
        self._switcher.sig_search_text_available.connect(
            self._handle_switcher_results)
```

There's no need to expose these methods as part of the public API of Projects, so they should be private.